tap/api/api.go

```diff
@@ -44,6 +44,15 @@ type Extension struct {
 	Dissector Dissector
 }
 
+type Capture string
+
+const (
+	Pcap    Capture = "pcap"
+	Envoy   Capture = "envoy"
+	Linkerd Capture = "linkerd"
+	Ebpf    Capture = "ebpf"
+)
```

Comment:
Add `UndefinedCapture Capture = ""` in order to handle uninitialized `Capture` variables.

Reply:
@nimrod-up9 done with https://github.com/up9inc/mizu/commit/2cbc54036a47c333012092c3146d46d92b87dcf2